### PR TITLE
Allow a chat to be muted indefinitely 

### LINKED ISSUE #719: Add support for muting chat forever.
As of now whatsapp-web.js can mute a chat until a specified time only. So I would like to suggest that add support for muting a chat forever if timestamp is null or defined.

### PR DESCRIPTION
**Pull request by purpshell**

### CODE REVIEW

**Review by tuyuribr — APPROVED** [review]
LGTM!

**Review by pedroslopez — CHANGES_REQUESTED** [review]
Thanks for adding this! I think we still need to divide by 1000

image

File: src/Client.js [hunk 1]
```diff
@@ -750,15 +750,16 @@ class Client extends EventEmitter {
...
-     * @param {Date} unmuteDate Date when the chat will be unmuted
+     * @param {?Date} unmuteDate Date when the chat will be unmuted, leave as is to mute forever
      */
-    async muteChat(chatId, unmuteDate) {
+    async muteChat(chatId, unmuteDate = undefined) {
+        unmuteDate = unmuteDate ? unmuteDate.getTime() : undefined;
         await this.pupPage.evaluate(async (chatId, timestamp) => {
             let chat = await window.Store.Chat.get(chatId);
             await chat.mute.mute(timestamp, !0);
-        }, chatId, unmuteDate.getTime() / 1000);
```
[the remark this answers is not in the document]
Comment (PurpShell) [reply src/Client.js:761]: hmm ok

**Comment by Kristrue** [comment]
I would love to know more about the conversation and yes I agree to mute

**Comment by PurpShell** [comment]
> Thanks for adding this! I think we still need to divide by 1000
> 
> image
> 

All done

**Review by pedroslopez — APPROVED** [review]

**Pull request merged**